### PR TITLE
#428 - core-lz setters.yaml project number needs derivation

### PR DESCRIPTION
**Pull request by obriensystems**
see test results in #428

### CODE REVIEW

**Review by cartyc — DISMISSED** [review]
/lgtm

File: solutions/core-landing-zone/setters.yaml [hunk 1]
```diff
@@ -47,9 +47,11 @@ data:
   ##########################
   #
   # This is the project where the config controller instance is running
-  # Values can be viewed in the Project Dashboard
+  # Values can be viewed in the Project Dashboard via https://console.cloud.google.com/welcome?project=$PROJECT_ID under "Project number"
```
Comment (alaincormier-ssc) [inline solutions/core-landing-zone/setters.yaml:50]: might be worth adding project ID also in the comment? 
`... under "Project ID" and "Project number"`

Comment (fmichaelobrien) [reply solutions/core-landing-zone/setters.yaml:50]: Alain, good point, will do, repatch eta 2-4h

Comment (obriensystems) [reply solutions/core-landing-zone/setters.yaml]: add reference to export block for PROJECT_ID in the readme
https://github.com/GoogleCloudPlatform/pubsec-declarative-toolkit/blob/main/docs/landing-zone-v2/README.md#initial-organization-configuration
and management-proiject-id below this line

**Review by cartyc — APPROVED** [review]

**Review by alaincormier-ssc — APPROVED** [review]

**Pull request merged**